portfolio/src/main/java/com/google/sps/servlets/data/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+/** An item on a todo list. */
+public final class Comment {
+
+  private final long id;
+  private final String comment;
+  private final String user;
+  private final long timestamp;
+
+  public Comment(long id, String comment, String user, long timestamp) {
```

Comment:
As this is a public method, try to give some comments for it. For example, what are these parameters.